plugins/extractit/icon_extractit/util/util.py

```diff
@@ -1,5 +1,5 @@
 class Regex:
-    Domain = r"(?:(?=[\p{L}\p{N}\p{Lo}\p{So}-]{1,63}\.)[\p{L}\p{N}\p{Lo}\p{So}]+(?:-[\p{L}\p{N}\p{Lo}\p{So}]+)*\.)+[\p{Ll}\p{Lo}]{2,63}(?:@|\/[\p{L}\p{N}@:%_\-+.~]*)?"
+    Domain = r"(?:(?=[\p{L}\p{N}\p{Lo}\p{So}-]{1,63}\.)[\p{L}\p{N}\p{Lo}\p{So}]+(?:-[\p{L}\p{N}\p{Lo}\p{So}]+)*\.)+[\p{Ll}\p{Lo}]{2,63}(?:@|=|\/[\p{L}\p{N}@:%_\/\-+?=#&.~]*)?"
```

Comment:
Are you sure this regex works? I tried it in regex101 with Python regex and it was not picking up any domains.

Reply:
I ran a few more tests and the results are correct. Unit testing for the domain extractor is also passing, so surely this regex works. I also checked this in regex101 with the Python regex and it looks like it doesn't support using Unicode categories like \p{L} in the regex and therefore doesn't find any domains.